infra/scripts/focused-test.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+# Copyright 2016 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Runs go test on any packages that differ from vmware/master that have
+# Go test files.
+#
+# Usage: infra/scripts/focused-test.sh
+#
+
+echo "Finding modified packages with test files"
+PKG=$(git diff --stat vmware/master --name-only | xargs dirname | uniq | while read i; do
```

Comment:
This assumes everyone is using the same remote name, `vmware`, for GitHub. That doesn't seem like a universally safe assumption. (I suspect some people stick with `origin`.)

Are there cases where diffing against `master` would incorrect?